index.js

```diff
@@ -389,7 +389,7 @@ class MongoModels {
         let filter;
 
         try {
-            filter = { _id: this._idClass(id) };
+            filter = { _id: this._idClass(id) , $isolated : 1 };
```

Comment:
I feel like these should be optional. I also realize in this case, there isn't a great place to add `$isolated` like the others do with the `filter` object. Thoughts on how we can make it an option?